libs/vanilla/src/inputs/radio.input.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  defineTransformView,
+  defineView,
+  ViewComponentProps,
+  ViewDefinition,
+} from '@reactlit/core';
+import { DetailedHTMLProps } from 'react';
+
+export type RadioInputProps<T> = Omit<
+  DetailedHTMLProps<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  >,
+  'value' | 'disabled'
+> & {
+  data: T[];
+  label?: string | React.ReactNode;
+  containerClassName?: string;
+  format?: (value: T) => string | React.ReactNode;
+  valueof?: (value: T) => string;
+  disabled?: string[] | ((value: T) => boolean);
+};
+
+export const RadioInputComponent = <T,>({
+  value,
+  stateKey,
+  setValue,
+  onChange,
+  data,
+  containerClassName,
+  format,
+  valueof,
+  disabled,
+  label,
+  ...props
+}: RadioInputProps<T> & ViewComponentProps<string | T>) => {
+  return (
+    <div className="flex gap-2 items-center">
+      {label && <label htmlFor={stateKey}>{label}</label>}
+      <div className={containerClassName}>
```

Comment:
What was the thinking on having the containerClassName here instead of or also on the top-level div? Folks sometimes could want to customize the placement of the label too relative to the data.